examples/example_O2_PES_PAD.py

```diff
@@ -117,7 +117,7 @@ def PAD(theta, beta, amp):
                     wspace=0.35, hspace=0.37)
 
 # Save a image of the plot
-plt.savefig("example_O2_PES_PAD.png", dpi=100)
+plt.savefig("output/example_O2_PES_PAD.png", dpi=100)
```

Comment:
While I kinda like the idea of hiding the basis sets from the user in the "bases" folder, I am not sure if I see why we would want to hide the example output? Shouldn't we want this to be obvious and easy to find?